scripts/bert/embedding.py

```diff
@@ -85,13 +71,13 @@ def __init__(self, ctx=mx.cpu(), model='bert_12_768_12',
         self.bert, self.vocab = gluonnlp.model.get_model(model,
                                                          dataset_name=dataset_name,
                                                          pretrained=True, ctx=self.ctx,
-                                                         use_pooler=True,
+                                                         use_pooler=False,
                                                          use_decoder=False,
                                                          use_classifier=False)
 
     def embedding(self, sentences, oov_way='avg'):
```

Comment:
would it be more convenient to offer `__call__` as an alias for this?

Reply:
`result = bert.embedding(sents, oov_way=args.oov_way)` would become
`result = bert_embedding(sents, oov_way=args.oov_way)`

Reply:
Yeah. Added in 8bdbc62